### PR TITLE
Do you know why programmers confuse halloween and christmas ?

### PR DESCRIPTION
**Pull request by Carreau**
because 31dec == 25oct.

**Do not change the date of your computer to test** it break all your current login, and certificate that suddenly appears as expired.

Also I don't know how, but I would like to enable these by default on said dates.

### CODE REVIEW

File: src/main/menu.js [hunk 1]
```diff
@@ -130,6 +130,38 @@ export const cell = {
...
+        {
+          label: 'Hohoho',
+          click: createSender('menu:theme', 'christmas'),
+        })
+} else if ( month == 10 && day==31){
+  theme_menu.push( {
+          label: 'Mwaaahahahhah',
+          click: createSender('menu:theme', 'halloween'),
+        })
+}
```
Comment (rgbkrk) [inline src/main/menu.js:162]: Oh my.

**Comment by rgbkrk** [comment]
I'm laughing so hard about this.

File: static/styles/theme-undefined.css [hunk 2]
```diff
@@ -21,32 +21,31 @@
...
-  20%   { background-color: #3498db; }
-  40%   { background-color: #f1c40f; }
-  60%   { background-color: #1abc9c; }
-  80%   { background-color: #e67e22; }
-  100% { background-color: #9b59b6; }
+  --cm-hint-bg-active: #E87D7D;
 }
 
 body {
-  animation: disco-mode 5s infinite;
```
Comment (rgbkrk) [inline static/styles/theme-undefined.css:51]: whoa whoa whoa, what are you doing to disco mode?!? 😝

Comment (Carreau) [inline static/styles/theme-undefined.css:51]: Ooops.

Comment (Carreau) [inline static/styles/theme-undefined.css:51]: I was testing with theme undefined and committed the file by mistake. Should be fix. 
My themes likely need some updates.

**Comment by Carreau** [comment]
Trying to think of other dates/weeks that could be fun to add. 
Definitively april 1st.

**Comment by Carreau** [comment]
And May the 4th. Can we play sounds in electron ?

**Comment by Carreau** [comment]
Note, pumkin from https://www.iconfinder.com/icons/60817/halloween_pumpkin_icon (free for commercial use) might still  want to have SO redesign one.

**Comment by rgbkrk** [comment]
Anything you can do in a browser, you can do in Electron. Better yet, you're able to load them direct from disk. :smile: Personally, I'd aim for just a couple of these for now until the rest of our platform is fairly stable.

**Comment by Carreau** [comment]
> Can we play sounds in electron ?

That was a rhetorical question.

**Comment by rgbkrk** [comment]
screen shot 2016-05-17 at 10 06 21 pm
screen shot 2016-05-17 at 10 06 36 pm

File: src/main/menu.js [hunk 3]
```diff
@@ -130,6 +130,38 @@ export const cell = {
...
+          label: 'Dark',
+          click: createSender('menu:theme', 'dark'),
+        },
+        {
+          label: 'Classic',
+          click: createSender('menu:theme', 'classic'),
+        },
+]
+
+const today = new Date();
```
Comment (rgbkrk) [inline src/main/menu.js:148]: To try this out, I set

```
const today = new Date('December 25, 2015');
```

directly in source

**Comment by Carreau** [comment]
Linting fixed, and introduced : `npm run lint:fix`

**Comment by rgbkrk** [comment]
> That was a rhetorical question.

Hahahaha.

**Comment by rgbkrk** [comment]
screen shot 2016-05-17 at 10 10 49 pm

The pumpkin stays fixed in one spot, I was kind of expecting it to hang out in the corner.

**Comment by Carreau** [comment]
Hum, weird.

**Comment by rgbkrk** [comment]
> Linting fixed, and introduced : npm run lint:fix

Well done!

File: src/main/menu.js [hunk 4]
```diff
@@ -183,24 +214,12 @@ export const view = {
...
-        {
-          label: 'Dark',
-          click: createSender('menu:theme', 'dark'),
-        },
-        {
-          label: 'Classic',
-          click: createSender('menu:theme', 'classic'),
-        },
-      ],
+      submenu: theme_menu,
```
Comment (rgbkrk) [inline src/main/menu.js:217]: 👍 glad you extracted this in general

**Pull request merged**